config/spring19_seminar/cola_var/sample.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#SBATCH --time=12:00:00
+#SBATCH --mem=20000
+#SBATCH --gres=gpu:1
+#SBATCH --job-name=myrun
+#SBATCH --output=slurm_%j.out
+
+python main.py --config_file config/spring19_seminar/bert.conf \
+    --overrides "exp_name = npi_bert, run_name = run_bert_mnli_cola, target_tasks = "cola", pretrain_tasks = "mnli", do_pretrain = 1" 
+
+python main.py --config_file config/spring19_seminar/bert.conf \
+    --overrides "exp_name = npi_bert, run_name = run_bert_mnli_colax, target_tasks = "colax", pretrain_tasks = "mnli", do_pretrain = 1" 
```

Comment:
Where's the code for the colax task? We'd have to check that in before checking this in.

Reply:
Now they're there. Simply run each thing from scratch.

Reply:
I added these lines into my local version. Not sure if this is helpful --- if we have a long list of cola-like tasks, all we need to do then is extend ALL_COLAX_TASKS

in "src/tasks/__init__.py"
ALL_COLAX_TASKS = ['colax']
ALL_GLUE_TASKS = ['sst', 'cola', 'mrpc', 'qqp', 'sts-b',
                  'mnli', 'qnli', 'rte', 'wnli', 'mnli-diagnostic'] + ALL_COLAX_TASKS

in "tasks.py"
from .__init__ import ALL_COLAX_TASKS
for colax in ALL_COLAX_TASKS:
    CoLATask = register_task(colax, rel_path='CoLA/%s/' % colax)(CoLATask)

Reply:
It doesn't really make sense to share/check-in the configs without this code. Go ahead and add it! Though I'd suggest something more informative than 'colax'. Maybe 'cola-npi-diagnostic'?

Reply:
Just added! Called them cola_npi_{some environment name}